docs/how-to/integration_test_allure.md

```diff
@@ -0,0 +1,64 @@
+# Allure Report Integration
+
+[Allure Report](https://allurereport.org/) for [integration_test_run.yaml](https://github.com/canonical/operator-workflows?tab=readme-ov-file#integration-test-workflow-canonicaloperator-workflowsgithubworkflowsintegration_testyamlmain)
+
+In order to integrate Allure with your repository, perform the following actions:
+
+## 1. Adding allure-pytest and pytest collection plugin
+
+Please add the following into the `requirements.txt` that is called by the integration test -
+
+```
+allure-pytest>=2.8.18
+```
+
+Add the following line under the dependencies (`deps`) in the integration section inside `tox.ini` -
+
+```
+git+https://github.com/canonical/operator-workflows@main\#subdirectory=python/pytest_plugins/allure_pytest_collection_report
+```
+
+## 2. Calling the allure-workflow
+
+Add the following lines at the end of the workflow that runs the integrations tests by calling the reuable workflow [integration-test.yaml](https://github.com/canonical/operator-workflows/blob/main/.github/workflows/integration_test.yaml):
+
+```
+  allure-report:
+    if: always() && !cancelled()
+    needs:
+      - [list of jobs with tests you would like to visualize]
+    uses: canonical/operator-workflows/.github/workflows/allure_report.yaml@main
+```
+
+Here's an [example for the above](https://github.com/canonical/github-runner-operator/pull/412).
```

Comment:
Nitpick: I would rephrase this a little differently. Something like `For an example of this implementation, see [the GitHub runner repository](https://github.com/canonical/github-runner-operator/pull/412).`

Reply:
done